src/RadialMenu/RadialMenu/PassThroughView.cs

```diff
@@ -6,13 +6,18 @@ namespace DK.Ostebaronen.Touch.RadialMenu
 {
     internal class PassThroughView : UIView
     {
+        public delegate void PointInsideDelegate(CGPoint point, UIEvent uievent);
```

Comment:
Can you please make a `EventArgs` class instead and use `event EventHandler<YourEventArgs>`